packages/web-app-files/src/components/SideBar/Links/PublicLinks/LinkEdit.vue

```diff
@@ -256,8 +255,7 @@ export default {
       }
 
       const days = parseInt(this.$_expirationDate.days, 10)
-
-      return addDays(new Date(), days)
+      return DateTime.now().setLocale(this.$language.current).plus({ days: days }).toJSDate()
```

Comment:
When key and value are the same in an object, you can use the shorthand version, in this example it's just `{ days }`